workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -133,9 +140,89 @@ private static DownloadResult Download()
                 EditorApplication.UnlockReloadAssemblies();
             }
 
+            AssetDatabase.Refresh();
+            SetPluginsCompatibility();
```

Comment:
Small optimisation - these can be done conditionally on the result of the download

Reply:
(y)